### ISSUE TITLE
`@enable 'serve *'` does nothing -- documentation disparity?

### ISSUE THREAD

**Issue by namuol** [mauricemach/zappa#113]
Without digging too deeply into recent changelists, it looks like using `@enable 'serve jquery'` or similar does nothing.

The "PicoChat" example still works, but only because it runs the `@client` method at some point, which -- I realized after looking at `zappa.coffee` -- causes `@use 'zappa'` to be called, which in turn seems to automatically do what `@enable 'serve jquery', 'serve sammy'` would do (among other things).

To see what I mean, try running the following app:

```
require('zappa') ->
  @enable 'serve jquery'
```

Then, try visiting http://localhost:3000/zappa/jquery.js -- no dice.

However, this works:

```
require('zappa') ->
  @use 'zappa'
```

Now, http://localhost:3000/zappa/jquery.js yields what we expect.

So as a workaround, I just `@use 'zappa'` in all my apps that need jquery or sammy, but the real fix might involve some minor code changes or a documentation update.

**Comment by TylerLH** [comment]
Thanks for noting this. I just ran into the same issue.

**Comment by iangreenleaf** [comment]
Yeah, I think it's a documentation change that's needed. Zappa is now bundling some connect middleware to serve the vendor javascript, and it looks like an intentional change on @mauricemach's part.
